pkg/cmd/types.go

```diff
@@ -110,17 +110,26 @@ func (i ignoredFields) Contains(field string) bool {
 var defaultIgnored = ignoredFields{"password", "token", "url", "uri", "name", "type", "id"}
 
 //Convert a kubernetes secret to a mobile.ServiceConfig
-func (dsc defaultSecretConvertor) Convert(id string, params map[string]string) (*ServiceConfig, error) {
+func (dsc defaultSecretConvertor) Convert(id string, params map[string]string, configType string) (*ServiceConfig, error) {
 	config := map[string]interface{}{}
 	headers := map[string]string{}
 	for k, v := range params {
 		if !defaultIgnored.Contains(k) {
-			config[k] = string(v)
+			if configType == "json" {
```

Comment:
we should check for the param named "config" as we are setting this we know it will always be called this. If it is not named this then we can just use the string value for the key

Reply:
Done!